src/bp/ui-admin/src/telemetry/client.ts

```diff
@@ -0,0 +1,121 @@
+import axios from 'axios'
+import { lang } from 'botpress/shared'
+import _ from 'lodash'
+import moment from 'moment'
+import ms from 'ms'
+import uuid from 'uuid'
+
+import store from '../store'
+
+interface EventPackageInfoType {
+  [key: string]: {
+    timeout: string
+    getPackage: Function
+  }
+}
+
+export const axiosConfig = {
+  baseURL: window.TELEMETRY_URL,
+  headers: {
+    withCredentials: false
+  }
+}
+
+const dataSchemaVersion = '1.0.0'
+
+const telemetrySchemaVersion = '1.0.0'
+
+const pathsInReduxTracked: string[] = ['user.profile.email', 'version.currentVersion', 'license.licensing.isPro']
+
+const eventPackageInfo: EventPackageInfoType = {}
```

Comment:
Get rid of "addTelemetryEvent". This would be simpler: 
```js
const telemetryEvents = {
  'ui-language': {
    expiry: '8h',
    data: { 
      return {
        user: {
        email: store.getState().user.profile.email,
        timezone: Intl.DateTimeFormat().resolvedOptions().timeZone
       },
    
}
}
```